### ISSUE TITLE
Display HTTP error after clicking Open In Portal

### ISSUE THREAD

**Issue by Jackymancs4** [makeworld-the-better-one/amfora#81]
Hello @makeworld-the-better-one 
While trying to diagnose #77 I changed the http config value to something invalid.

After that I forgot to change it back to `default`, and while navigating some gemini pages I tried to open a png image with the portal option. It silently failed and only after a while I tried to open another http page that correctly prompted the error (which reminded me to fix the config).

So, in summary, if there aren't any technical reasons against it, I think it's a good idea to enable modal responses (both error and info) after a `That file could not be displayed. What would you like to do?` interaction.

**Comment by makeworld-the-better-one** [comment]
Thanks for finding this, the error popup should definitely occur in this case as well.
